### ISSUE TITLE
Retrospective Board Summary: session date = current date, not board date in the past

### ISSUE THREAD

**Issue by kw80** [microsoft/vsts-extension-retrospectives#515]
The session date on the "Retrospective Board Summary" is the current date and not the start date of the selected board.

**Steps to reproduce**
- Azure Devops: Go to Boards > Retrospectives
- Select an old release board
- Click on the elipsis ("...") button
- Select "Show Retrospective Board Summary"

**Expected Result**
Session date is the start date of the selected board, which is a date in the past for old release boards

**Actual Result**
Session date is the current date

See line: https://github.com/microsoft/vsts-extension-retrospectives/blob/main/RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx#L1625
`Session date: {new Intl.DateTimeFormat('en-US', { year: 'numeric', month: 'short', day: 'numeric' }).format(this.state.currentBoard.startDate)}</div>`

**Comment by dphanson63** [comment]
Yes, please fix this.  Not sure if it's a new issue, but I also just noticed this yesterday.

I'd be okay if the Session Date was the date the board was created or the date of the last entry, but displaying current date was useless, when I was looking back at past retro participation and team assessment stats.
